src/Component/Nav/NavScroll.js

```diff
@@ -0,0 +1,201 @@
+import React, { useState, useEffect } from 'react'
+import styled, { StyleSheetManager } from "styled-components";
+import Dropdown from './Dropdown';
+import { AiOutlineDownload } from 'react-icons/ai';
+import { FaUser } from 'react-icons/fa';
+import { FaHeart } from 'react-icons/fa';
+import { AiFillShopping } from 'react-icons/ai';
+import { RiLoginBoxFill } from 'react-icons/ri';
+import { FaSearch } from 'react-icons/fa';
+
+export default function NavScroll(props) {
+  const [navData, setNavData] = useState([]);
+  const [isHover, setIsHover] = useState(false);
+
+  const navMenuData = navData[0]?.data;
+  const navDetailMenuL = navData[1]?.data;
+  const navDetailMenuR = navData[2]?.data;
+
+  useEffect(() => {
+    fetch("/data/navData.json")
+      .then((response) => response.json())
+      .then((response) => {
+        setNavData(response.navData)
+      })
+  }, []);
+
+  const openDropDown = () => {
+    setIsHover(true);
+  }
+  
+  const closeDropDown = () => {
+    setIsHover(false);
+  }
```

Comment:
이렇게 간다한 건 한 줄로 줄여도 될 거 같네요!
```suggestion
  const openDropDown = () => setIsHover(true);
  
  const closeDropDown = () => setIsHover(false);
```